# an very old patch cherry-picked in every glib-2.0 patch directory. The earliest container
# for it is 2.6.5 in OE. The earliest commit for it is c8e5702127e507e82e6f68a4b8c546803accea9d
# in OE side which ports from previous bitkeeper SCM. In OE side it's only used til 2.12.4.
#
# keep it since it's always cleaner to not hardcode destination path. Use @datadir@ is more
# portable here. mark for upstream
#
# by Kevin Tian <kevin.tian@intel.com>, 06/25/2010
--- glib-2.10.3/glib-gettextize.in.old	2006-06-21 12:46:45.000000000 +0100
+++ glib-2.10.3/glib-gettextize.in	2006-06-21 12:48:14.000000000 +0100
@@ -49,7 +49,7 @@
     ;;
 esac
 
-gettext_dir=$prefix/share/glib-2.0/gettext
+gettext_dir=@datadir@/glib-2.0/gettext
 
 while test $# -gt 0; do
   case "$1" in
